core/src/com/unciv/models/ruleset/Speed.kt

```diff
@@ -56,7 +56,7 @@ class Speed : RulesetObject(), IsPartOfGameInfoSerialization {
 
     override fun getUniqueTarget(): UniqueTarget = UniqueTarget.Speed
 
-    override fun makeLink(): String = "GameSpeed/$name"
+    override fun makeLink(): String = "Speed/$name"
```

Comment:
I have a suggestion - since these are linked to the Enum of the page, we can add a 'fun `makeLink(CivilopediaCategory:Category)` in ICivilopedia and use that, instead of this manual string stuff.

But probably a different PR.